source/js/buyers-guide/product-tab.js

```diff
@@ -12,18 +17,34 @@ function setHighlight(target) {
   return target;
 }
 
+/**
+ * Remove highlight from the given tab by updating its class list and aria attribute
+ * @param {*} target tab to be unhighlighted
+ * @returns {HTMLElement} unhighlighted tab with updated class list and aria attribute
+ */
 function removeHighlight(target) {
   let button = target.querySelector("button");
   button.classList.remove("tw-bg-black", "tw-text-white");
   button.classList.add("tw-bg-white", "tw-text-black", "hover:tw-text-blue-80");
+  button.setAttribute("aria-selected", "false");
```

Comment:
This line was missing. In `function setHighlight(target) { ... }` above we do `button.setAttribute("aria-selected", "true");`. Here in `function removeHighlight` we should set the aria attribute to false.